travis-ymls/ginkgo-travis.yml

```diff
@@ -0,0 +1,43 @@
+# ----------------------------------------------------------------------------
+#
+# Package             : ginkgo
+# Source Repo         : https://github.com/onsi/ginkgo
+# Travis Job Link     : https://travis-ci.com/github/santosh653/ginkgo
+# Created travis.yml  : No
+# Maintainer          : Santosh Kulkarni <santoshkulkarni70@gmail.com>
+#
+# Script License      : Apache License, Version 2 or later
+#
+# ----------------------------------------------------------------------------
+
+arch:
+  - amd64
+  - ppc64le
+language: go
+go:
+  - 1.14.x
+  - 1.15.x
+  - tip
+jobs:
+ exclude:
+  - go: 1.14.x
+    arch: ppc64le
```

Comment:
Reason for excluding 1.14.x? @santosh653

Reply:
@Siddhesh-Ghadi This is the repeated PR(I have closed the previous one) I raised after I was not sure how to squash the commits on already raised PR. 
In the earlier PR, Priya had asked me to skip the failure(As there was a failure on ppc64le for go:1.14.x) & run the code for rest, accordingly I have raised new PR with suggested changes.
Kindly let me know if you need more details.,

Thank You!!

Reply:
In future PRs make sure to add a comment(reason) in the file, if you are excluding a particular job.